src/main/java/com/capgemini/setrack/controller/BookingController.java

```diff
@@ -0,0 +1,35 @@
+package com.capgemini.setrack.controller;
+
+import com.capgemini.setrack.model.Booking;
+import com.capgemini.setrack.repository.BookingRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/booking/")
+public class BookingController {
+
+    @Autowired
+    private BookingRepository bookingRepository;
+
+    @RequestMapping(value = "", method = RequestMethod.GET)
+    public Iterable<Booking> getAllBookings(){
+        return this.bookingRepository.findAll();
+    }
+
+    @RequestMapping(value = "create", method = RequestMethod.POST)
+    public Booking createBooking(@RequestBody Booking booking){
+        this.bookingRepository.save(booking);
+        return booking;
+    }
+
+    @RequestMapping(value = "delete/{id}", method = RequestMethod.DELETE)
+    public void deleteBooking(@PathVariable long id) {
```

Comment:
Dit anders oplossen via delete-functionaliteit die spring/jpa biedt. Geen reden meer om nu alles op te halen.

Reply:
Goed idee! Ik heb het ook gelijk aangepast in de GuestController en RoomController!